.github/workflows/rebaseline-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Rebaseline Tests
+
+on: [workflow_dispatch]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  rebaseline-tests:
+    name: Rebaseline Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: pip install
+        run: |
+          which python3
+          python3 --version
+          python3 -m pip install -r requirements-dev.txt
+      - name: Install emsdk
+        run: |
+          EM_CONFIG=$HOME/emsdk/.emscripten
+          echo $EM_CONFIG
+          curl -# -L -o ~/emsdk-main.tar.gz https://github.com/emscripten-core/emsdk/archive/main.tar.gz
+          tar -C ~ -xf ~/emsdk-main.tar.gz
+          mv ~/emsdk-main ~/emsdk
+          cd ~/emsdk
+          ./emsdk install tot
+          ./emsdk activate tot
+          echo "JS_ENGINES = [NODE_JS]" >> $EM_CONFIG
+          echo "final config:"
+          cat $EM_CONFIG
+          echo "EM_CONFIG=$EM_CONFIG" >> $GITHUB_ENV
+      - name: Rebaseline tests
+        run: |
+          ./bootstrap
+          git config user.name emscripten-bot
+          git config user.email emscripten-bot@users.noreply.github.com
+          ./tools/maint/rebaseline_tests.py
+      - name: Create PR
+        id: cpr
+        run: |
+          git push origin
+          gh pr create --fill
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think there is a `secrets.GITHUB_TOKEN`? (unless it's just the default token, in which case I don't think you need to specify it manually?). You can use `secrets.EMSCRIPTEN_BOT_TOKEN` to authenticate as the same account we use for the release automation. But I'm not sure it's necessary; I think the default token has enough permissions.

Reply:
I think this is the token that is used when a user runs the action explicitly.  I think it should work for this case since I initially just want to trigger it manually.

Reply:
Right but what I'm saying is that I think there's an ambient token that just gets passed to all the steps by default. You can see it e.g. in the "set up job" phase of the run: https://github.com/emscripten-core/emscripten/actions/runs/13597808170/job/38018423393 I don't think explicitly passing it actually does anything.